install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
```

Comment:
This script looks like it should be run with `sudo` but no `sudo` permission verification was made at the start of the script.